QUEST_LV_0100.tsv

```diff
@@ -2390,11 +2390,11 @@ QUEST_LV_0100_20150317_002389	$I can do that
 QUEST_LV_0100_20150317_002390	$The Road to Klaipeda (2)
 QUEST_LV_0100_20150317_002391	$I'll lend my strength to you
 QUEST_LV_0100_20150317_002392	$Enter Klaipeda
-QUEST_LV_0100_20150317_002393	$To the Battle Commander (1)
+QUEST_LV_0100_20150317_002393	$Talk to Battle Commander (1)
```

Comment:
I intentionally left it as is. If I put the "the" in it, then the UI breaks. Will post a screenshot on discord.